filters/auth/oidc.go

```diff
@@ -28,6 +30,7 @@ const (
 	oauthOidcCookieName = "skipperOauthOidc"
 	stateValidity       = 1 * time.Minute
 	oidcInfoHeader      = "Skipper-Oidc-Info"
+	cookieMaxSize       = 4069
```

Comment:
I searched a bit and indeed it's everywhere, but no documentation, that explicitly states this.
It would be nice to add a note `// known max cookie size for common browsers` or something like that

Reply:
I've verified this assumption using this service: http://browsercookielimits.squawky.net/